pkg/api/steve/proxy/proxy_test.go

```diff
@@ -41,21 +43,28 @@ func TestLocalCluster(t *testing.T) {
 		desiredResponseMessage string
 	}{
 		{
-			name:                   "basic test case, no match",
+			name:                   "no matching path",
 			requestPath:            "/v1/not/a/path",
 			userCanAccessLocal:     false,
 			desiredResponseCode:    defaultResponseCode,
 			desiredResponseMessage: defaultResponseMessage,
 		},
 		{
-			name:                   "management crds with local cluster access",
+			name:                   "management disallowed crd with local cluster access",
 			requestPath:            "/apis/management.cattle.io/v3/test-crd",
 			userCanAccessLocal:     true,
 			desiredResponseCode:    defaultResponseCode,
 			desiredResponseMessage: defaultResponseMessage,
 		},
 		{
-			name:                   "management crds without local cluster access",
+			name:                   "management allowed crd with local cluster access",
+			requestPath:            "/apis/management.cattle.io/v3/podsecurityadmissionconfigurationtemplates",
+			userCanAccessLocal:     false,
```

Comment:
Is this right for this test?

Reply:
`podsecurityadmissionconfigurationtemplates` is in the AllowAll set, so it should be exempt from auth handlers.

Reply:
The name says "with local cluster access" but then `userCanAccessLocal` is false

Reply:
Fixed!